exercises/src/test/scala/fpinscala/LazinessSpec.scala

```diff
@@ -34,6 +36,9 @@ class LazinessSpec extends AnyFlatSpec with Matchers {
     Stream(1, 2, 3, 4).forAll(_ > 0) shouldBe true
     Stream.cons(1, Stream.cons(2, Stream.cons(-3, {throw new Exception(" :( ")}))).forAll(_ > 0) shouldBe false
     Stream(1, 2, -3, 4).forAll(_ > 0) shouldBe false
+
+    def nat(start: Int): Stream[Int] = Stream.cons(start, nat(start + 1))
+    nat(1).forAll(_ % 10 != 0) shouldBe false
```

Comment:
You can use `Stream.from` that you wrote instead, it's the same thing as `nat`, I just hadn't seen it when I wrote the comment that defined `nat`.